docker-compose.yaml

```diff
@@ -1,32 +1,32 @@
-version: '2'
+version: '3.2'
 
 services:
-  service_php:
+  php:
     build: docker/php
-    tty: true
-    labels:
```

Comment:
same. But I could be wrong! Is there a reason for adding this `label`?

Reply:
the same : I took that from another symfony project